### ISSUE TITLE
Unhandled Rejection (TypeError): t.batchNormalization is not a function

### ISSUE THREAD

**Issue by MichaelFay** [justadudewhohacks/face-api.js#644]
I am getting the following error when trying to recreate the example code for bbt. 
Unhandled Rejection (TypeError): t.batchNormalization is not a function

My packages are: 
  "dependencies": {
    "@tensorflow-models/coco-ssd": "^0.1.1",
    "@tensorflow/tfjs": "^2.0.1",
    "@tensorflow/tfjs-core": "^2.0.1",
    "@tensorflow/tfjs-data": "^2.0.1",
    "@testing-library/jest-dom": "^4.2.4",
    "@testing-library/react": "^9.5.0",
    "@testing-library/user-event": "^7.2.1",
    "bootstrap": "^4.5.0",
    "capture-video-frame": "^0.1.3",
    "face-api.js": "^0.22.2",
    "react": "^16.13.1",
    "react-bootstrap": "^1.0.1",
    "react-bootstrap-range-slider": "^1.0.0",
    "react-dom": "^16.13.1",
    "react-player": "^2.3.0",
    "react-redux": "^7.2.0",
    "react-scripts": "3.4.1",
    "react-thunk": "^1.0.0",
    "react-webcam": "^5.1.0",
    "redux": "^4.0.5",
    "tslib": "^2.0.0",
    "typescript": "^3.9.5"
  },

**Comment by vladmandic** [comment]
face-api is not yet compatible with tfjs 2.0, you have to use older 1.7.x branch.

**Comment by juamanuel** [comment]
> face-api is not yet compatible with tfjs 2.0, you have to use older 1.7.x branch.

It worked for me with these versions.

```
"dependencies": {
    "@tensorflow/tfjs-node": "^1.7.0",
    "aws-sdk": "2.785.0",
    "canvas": "^2.6.1",
    "face-api.js": "^0.22.2",
    "gm": "1.23.1",
    "sharp": "0.26.3",
    "smartcrop-sharp": "2.0.3"
  }
```

**Comment by vladmandic** [comment]
since this thread is still alive, I might mention that I did a full port of `face-api` to tfjs 2.x (including updated models as well as entire toolchain and typescript and ecma standards). 
 
if you want, check out <https://github.com/vladmandic/face-api>

**Comment by juamanuel** [comment]
> since this thread is still alive, I might mention that I did a full port of `face-api` to tfjs 2.x (including updated models as well as entire toolchain and typescript and ecma standards).
> 
> if you want, check out https://github.com/vladmandic/face-api

Awesome!! thanks
